interface-definitions/dhcpv6-server.xml

```diff
@@ -278,25 +278,36 @@
                   <tagNode name="static-mapping">
                     <properties>
                       <help>Name of static mapping</help>
+                      <constraint>
+                        <regex>^[-_a-zA-Z0-9.]+$</regex>
```

Comment:
There is no need to preceed a regex with ^ as this is alsays done „under the hood“

Reply:
There are 5 other places in the file (and more in others) that also begin with ^, shouldn't they all be the same?

Reply:
I see, validate-value.py uses re.fullmatch so the ^ is implied. IMO it should be removed everywhere then and not just here, to be consistent.

Reply:
True, but that should be a separate commit then.

Reply:
removed https://github.com/vyos/vyos-1x/pull/72/commits/6b86cd54c12d34037d00667eaeaf2dc68d0a41a1